source/git-api.js

```diff
@@ -71,9 +71,8 @@ exports.registerApi = (env) => {
           });
         }
       }).then(() => {
-        const watcher = fs.watch(pathToWatch, options || {});
```

Comment:
Please use the file watcher events which I added. See  https://github.com/FredrikNoren/ungit/pull/1263#discussion_r369016772

Reply:
Ah.... why?  API documentation does not mention error event.

Reply:
https://nodejs.org/api/fs.html#fs_fs_watch_filename_options_listener

Reply:
The returned `fs.FSWatcher` does have the events: https://nodejs.org/api/fs.html#fs_class_fs_fswatcher
And it is important to listen for the error event otherwise the process would crash if the event isn't handled. This was a source of clicktest failures because we deleted the temp directory while the directory was still openend in ungit.

Reply:
You are right, I find this api design to be confusing...  multiple places to register events and when you register `change` event listener, it would multiple sub event types?  This is bit odd especially when `fswatcher{}` is not shared anywhere else.

Yeah I will fix it up later

Reply:
Good catch!